--- libobjc2-9999/CMakeLists.txt.orig	2013-06-25 15:56:16.186898073 +0200
+++ libobjc2-9999/CMakeLists.txt	2013-06-25 15:56:42.889967656 +0200
@@ -294,8 +294,8 @@
 set(GNUSTEP_INSTALL_TYPE ${DEFAULT_INSTALL_TYPE} CACHE STRING
 	"GNUstep installation type.  Options are NONE, SYSTEM, NETWORK or LOCAL.")
 if (${GNUSTEP_INSTALL_TYPE} STREQUAL "NONE")
-	SET(LIB_INSTALL_PATH "lib" CACHE STRING
-		"Subdirectory of the root prefix where libraries are installed.")
+	SET(LIB_INSTALL_PATH "${CMAKE_INSTALL_LIBDIR}" CACHE STRING
+		"Subdirectory of the root prefix where libraries are installed.")
 	SET(HEADER_INSTALL_PATH "include")
 else ()
 	EXEC_PROGRAM(gnustep-config
